src/components/Dashboard/FilingHistoryList/DetailsList.vue

```diff
@@ -18,15 +20,21 @@
 
     <!-- the detail comments list-->
     <v-list class="pb-0">
-      <v-list-item class="pl-0 pr-0 detail-body" v-for="(comment, index) in filing.comments" :key="index">
+      <v-list-item
+        v-for="(comment, index) in filing.comments"
+        :key="index"
+        class="pl-0 pr-0 detail-body"
+      >
         <v-list-item-content>
-          <v-list-item-title class="body-2">
+          <v-list-item-title class="text-body-2">
```

Comment:
What is this change?

Reply:
ESLint decided to add this. So, I went with it and fixed the test for it.

Reply:
OK. I see it got changed in a few places. So the class exists? And it looks the same as before on screen?

Reply:
Must be a Vuetify 3 vs Vuetify 2 thing.

We need a way to remember to change this when we do upgrade to Vuetify 3. Any ideas?

Reply:
Shall I add a comment in the template?
`<!-- FUTURE: Change body-2 (Vuetify 2) to text-body-2 (Vuetify 3) -->`

Reply:
My thoughts;
1. change it to a warning instead
or
2. leave a code comment to remove the rule
or
3. create a tech debt ticket for this, or add to an existing Vue/Vuetify 3 ticket

Reply:
> Shall I add a comment in the template?

Not in the template, as that would be in many places. In the config maybe?

Reply:
I like the first one! This way it'll be in our face whenever we run. This'll make sure that we remember once we do the upgrade.

Reply:
Haha, fine by me :)

Reply:
Sev, circling back this. If I have the rule as 'warn', whenever I run `npm run lint`, ESLint is changing this class. It's determined to change it. I guess I'll be turning off the rule and adding a code comment in the eslint config file.

Reply:
Haha, darn. I'm surprised there isn't a way to override that behaviour.

I'm OK with a comment in the config file.